src/Bicep.Core/FileSystem/FileResolver.cs

```diff
@@ -216,5 +216,10 @@ public bool TryDirExists(Uri fileUri)
             return fileUri.IsFile && Directory.Exists(fileUri.LocalPath);
         }
 
+        public bool TryFileExists(Uri fileUri)
```

Comment:
>TryFileExists [](http://example.com/codeflow?start=20&length=13)

Should be called `FileExists`. (I'm renaming the one above in my registry PR, so no need to match.)